scripts/build.py

```diff
@@ -146,6 +145,7 @@ def get_cache_slug():
     ASSETS_OUT_DIR = os.path.join(BUILD_DIR, 'assets', '')
     copy_files_source_to_target(ASSETS_SRC_DIR, ASSETS_OUT_DIR)
 
+    process_third_party_libs()
```

Comment:
Ah, I remember commenting on something related when I reviewed gulpfile.js in your last PR. Let me check...

Got it. So, we now have third_party/generated/... and third_party/generated/prod/..., and I remember that it seemed weird to do things this way. You explained why this was, but now I'm thinking that really the odd part is to have a third_party/generated/prod/... at all, since it causes unnecessary confusion (if there's a prod, shouldn't there be a dev)?

So -- just to double check, do we still need third_party/generated/prod for anything? If so, what do we need it for? If not, let's get rid of it, and instead pass the production cache slug into gulpfile.js as an argument from build.py (and anywhere else where the gulp script is called from). That will remove the extra directory and keep things cleaner. I think this is simpler than keeping track of an additional temporary directory that's never really used.